app/src/main/java/com/example/musicdemo/MainActivity.java

```diff
@@ -1,16 +1,65 @@
 package com.example.musicdemo;
 
+import android.content.ComponentName;
+import android.content.Intent;
+import android.content.ServiceConnection;
+import android.os.IBinder;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
 
-public class MainActivity extends AppCompatActivity {
+public class MainActivity extends AppCompatActivity implements View.OnClickListener {
 
+    private Button mButtonStart, mButtonStop;
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
-       ;
+        mButtonStart = (Button) findViewById(R.id.btn_start);
+        mButtonStop = (Button) findViewById(R.id.btn_stop);
+
+        mButtonStop.setOnClickListener(this);
```

Comment:
```suggestion
        findViewById(R.id.btn_stop).setOnClickListener(this);
```
hạn chế khai báo instance variable
nếu chỉ cần để bắt sự kiện có thể sử dụng ntn